docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md

```diff
@@ -0,0 +1,196 @@
+          cpu: "1"
+          memory: "1Gi"
+      replicas: 1
+      image: vesoft/nebula-metad
+      version: {{nebula.branch}} //Change the value from v2.5.1 to {{nebula.branch}}.
+      dataVolumeClaim:
+        resources:
+          requests:
+            storage: 2Gi
+        storageClassName: gp2
+      logVolumeClaim:
+        resources:
+          requests:
+            storage: 2Gi
+        storageClassName: gp2
+    storaged:
+      resources:
+        requests:
+          cpu: "500m"
+          memory: "500Mi"
+        limits:
+          cpu: "1"
+          memory: "1Gi"
+      replicas: 3
+      image: vesoft/nebula-storaged
+      version: {{nebula.branch}} //Change the value from v2.5.1 to {{nebula.branch}}.
+      dataVolumeClaim:
+        resources:
+          requests:
+            storage: 2Gi
+        storageClassName: gp2
+      logVolumeClaim:
+        resources:
+          requests:
+            storage: 2Gi
+        storageClassName: gp2
+    reference:
+      name: statefulsets.apps
+      version: v1
+    schedulerName: default-scheduler
+    imagePullPolicy: Always
+  ```
+
+3. Run the following command to apply the version update to the cluster CR.
+   
+  ```bash
+  kubectl apply -f apps_v1alpha1_nebulacluster.yaml
+  ```
+
+4. After waiting for about 2 minutes, run the following command to see the image versions of the services in the cluster have been changed to {{nebula.branch}}.
+   
+  ```bash
+  kubectl get pods -l app.kubernetes.io/cluster=nebula  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
+  ```
+
+  Output:
+
+  ```bash
+        1 vesoft/nebula-graphd:{{nebula.branch}}
+        1 vesoft/nebula-metad:{{nebula.branch}}
+        3 vesoft/nebula-storaged:{{nebula.branch}}  
+  ```
+
+## Upgrade a Nebula Graph cluster with Helm
+
+### Prerequisites
+
+You have created a Nebula Graph cluster with Helm. For details, see [Create a Nebula Graph cluster with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
+
+### Steps
+
+1. Update information of available charts locally from chart repositories.
+
+  ```bash
+  helm repo update
+  ```
+
+2. Set environment variables to your desired values.
+
+  ```bash
+  export NEBULA_CLUSTER_NAME=nebula         # The desired Nebula Graph cluster name.
+  export NEBULA_CLUSTER_NAMESPACE=nebula    # The desired namespace where your Nebula Graph cluster locates.
+  ```
+
+3. Upgrade a Nebula Graph cluster.
+
+  For example, upgrade a cluster to {{nebula.branch}}.
+
+  ```bash
+  helm upgrade "${NEBULA_CLUSTER_NAME}" nebula-operator/nebula-cluster \
+      --namespace="${NEBULA_CLUSTER_NAMESPACE}" \
+      --set nameOverride=${NEBULA_CLUSTER_NAME} \
+      --set nebula.version={{nebula.branch}}
+  ```
+
+  The value of `--set nebula.version` specifies the version of the cluster you want to upgrade to.
+
+4. After waiting for a while, run the following command to check the status and version of the upgraded cluster.
```

Comment:
```suggestion
4. Run the following command to check the status and version of the upgraded cluster.
```

`After waiting for a while` is of no help. What do you think?
Either we provide the output of a successful upgrade, or we let the users judge by themselves whether the upgrade is successful or not.

Reply:
exactly agree with that